acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TrustingSimpleHttpsClient.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.test.acceptance.dsl;
+
+import java.security.MessageDigest;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import okhttp3.OkHttpClient;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes;
+import tech.pegasys.teku.infrastructure.crypto.MessageDigestFactory;
+import tech.pegasys.teku.infrastructure.crypto.SecureRandomProvider;
+
+public class TrustingSimpleHttpsClient extends SimpleHttpClient {
+  private static final Logger LOG = LogManager.getLogger();
+  private static final TrustManager[] TRUST_ALL_CERTS =
```

Comment:
This probably isn't `TRUST_ALL_CERTS` now that it's specifically trusting our test cert.

Reply:
fair point...